ttl/ttlworker/task_manager.go

```diff
@@ -269,14 +281,22 @@ func (m *taskManager) rescheduleTasks(se session.Session, now time.Time) {
 		return
 	}
 
+loop:
 	for _, t := range tasks {
```

Comment:
Do we need to count the running tasks before locking a task to reduce some unnecessary queries?

Reply:
I don't think it's necessary.

There is only one extra transaction (count the running tasks also requires one query), becuase we'll `break loop` after encoutering `errTooManyRunningTasks`.